rails/db/migrate/20210618123809_remove_geni_models.rb

```diff
@@ -0,0 +1,16 @@
+class RemoveGeniModels < ActiveRecord::Migration[6.1]
+  def up
+    drop_table :geniverse_activities
+    drop_table :geniverse_articles
+    drop_table :geniverse_cases
+    drop_table :geniverse_dragons
+    drop_table :geniverse_help_messages
+    drop_table :geniverse_unlockables
+    drop_table :geniverse_users
```

Comment:
Because some developers and portals won't have these tables, this should work even if there are no tables. Does drop_table handle this case, or does it need to be wrapped with some exception handling?

Reply:
Looks like there is an option to check if the table exists:

https://apidock.com/rails/v5.2.3/ActiveRecord/ConnectionAdapters/SchemaStatements/drop_table

```
drop_table(table_name, options = {}) public
Drops a table from the database.

:force
Set to :cascade to drop dependent objects as well. Defaults to false.

:if_exists
Set to true to only drop the table if it exists. Defaults to false.
```